ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ipv6-prefix-lists.j2

```diff
@@ -1,10 +1,14 @@
-{# eos - prefix-lists #}
-{% if ipv6_prefix_lists is defined and ipv6_prefix_lists is not none %}
+{# eos - ipv6-prefix-lists #}
+{% if ipv6_prefix_lists is arista.avd.defined %}
 {%     for ipv6_prefix_list in ipv6_prefix_lists | arista.avd.natural_sort %}
 !
 ipv6 prefix-list {{ ipv6_prefix_list }}
-{%         for sequence in ipv6_prefix_lists[ipv6_prefix_list].sequence_numbers | arista.avd.natural_sort %}
+{%        if ipv6_prefix_lists[ipv6_prefix_list].sequence_numbers is arista.avd.defined %}
```

Comment:
Wrong indentation: missing space

Reply:
As `arista.avd.defined` is included in `arista.avd.natural_sort`, if statement can be removed